### PR TITLE
ci: Fix Minitest

### PR DESCRIPTION
**Pull request by arielvalentin**
Minitest 5.19 removes the automatic compatability layer that allowed users to use `MiniTest` camel case module name.

https://github.com/minitest/minitest/compare/v5.18.1...v5.19.0

Rather than continuing to support antiquated code lets bring ourselves up to date with a rename that happened almost 10 years ago.

### CODE REVIEW

**Review by scbjans — APPROVED** [review]

**Comment by arielvalentin** [comment]
@scbjans although I appreciate a review, I require one from @open-telemetry/ruby-contrib-approvers 

Have you reconsidered my offer to work towards approver? 😃

**Comment by scbjans** [comment]
@arielvalentin I know you need another official approval. 
My approval was my way of saying I tested and verified this PR (to fix CI for the open PR on the container resource detector).

**Review by plantfansam — APPROVED** [review]
❤️ ty!

**Pull request merged**
